examples/Console/TestJaegerExporter.cs

```diff
@@ -45,9 +45,10 @@ internal static object Run(string host, int port)
             // the reporoot\examples\Console\.
             // (eg: C:\repos\opentelemetry-dotnet\examples\Console\)
             //
-            // dotnet run jaeger -h localhost -p 6831
+            // dotnet run --project Examples.Console.csproj -- jaeger -h localhost -p 6831
```

Comment:
Would this work?

```suggestion
            // dotnet run -- jaeger -h localhost -p 6831
```

Reply:
@reyang it does work as well, yes. It seems appropriate to specify the project. Is there any issue with specifying the project as is done [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/examples/Console/Program.cs#:~:text=///%20dotnet%20run%20%2D%2Dproject%20Examples.Console.csproj%20%2D%2D%20jaeger%20%2Dh%20localhost%20%2Dp%206831)?

Reply:
I don't have strong opinion here, personally I prefer shorter/simpler version if the intention is clear (e.g. I feel repeating the project name here is a duplicated information given the project folder already provided the context).

Not a blocker though.